static/styles/rewards/claim-table.css

```diff
@@ -85,6 +85,8 @@ table td div {
   word-break: break-all;
   overflow: hidden;
   text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: 200px;
```

Comment:
Why 200px, is this the most optimal width given the available space across mobile and desktop?

Reply:
I changed it to 240px here is how it will look on an iPhone SE

<img width="1512" alt="Screenshot 2024-07-06 at 2 33 55 AM" src="https://github.com/ubiquity/pay.ubq.fi/assets/119438857/3cb57a86-034e-4760-bfc5-9b8e568653fd">

I think increasing it further may cause issues

Reply:
I do too but I also think this approach isn't very responsive and could be improved to cater for other viewports. While it may also resolve the other overlap I mentioned, it will likely make it harder to effectively size both.

You have the right idea mate it's just the getting the execution right now.

P.S I appreciate the images as will other reviewers but one after each edit is not required, you can keep one or two main images/videos in the first comment up to date when you are satisfied with your changes. The odd image throughout the PR is okay but it makes it easier for other reviewers to quickly grasp where you at with things if it's up the top. Thanks @Sadaf-A